### PR TITLE
Update templayed.js to v0.2.3

### PR DESCRIPTION
**Pull request by archan937**
This fixes the following cases:

* `"good || bad"`
  - It was rendered as: `"good  bad"`
  - With this fix as: `"good || bad"`

* `{{nul}} || 1` with variables `{nul: 0}`
  - It was rendered as: ` || 1`
  - With this fix as: `0 || 1`

### CODE REVIEW

File: functions/expression/1.0/index.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import templayed from 'templayed';
```
Comment (jwielinga) [inline functions/expression/1.0/index.js:1]: Just curious, do we want to limit expressions to a single function block? Or have it in a shared directory?

Comment (emiel-betty) [reply functions/expression/1.0/index.js:1]: If we gonna implement other blocks with this templated library then we should move it imo

Comment (archan937) [reply functions/expression/1.0/index.js:1]: What @emiel-betty says indeed

**Review by emiel-betty — APPROVED** [review]

**Review by jwielinga — COMMENTED** [review]
Just curious, do we want to limit expressions to a single function block? Or have it in a shared directory?

**Pull request merged**